### PR TITLE
refactor(sol): leverage @open-amt-cloud-toolkit/ui-toolkit-angular

### PR DESCRIPTION
**Pull request by Mahendra-intel**

### CODE REVIEW

File: src/app/devices/sol/sol.component.html [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
-<div class="container">
-    <div id="terminal" class="xtermDisplay" style="width: fit-content;"></div>
-</div>
\ No newline at end of file
+<app-device-toolbar
+  [deviceState]="deviceState"
+  (stopSol)="stopSol()"
+  [isLoading]="isLoading"
+></app-device-toolbar>
+<amt-sol  *ngIf="readyToLoadSol"
```
Comment (rsdmike) [inline src/app/devices/sol/sol.component.html:6]: missing deviceId, authtoken, mpsServer -- please update sol to have these passed in

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/app/devices/sol/sol.component.html (hunk 1 above)
Comment (Mahendra-intel) [reply src/app/devices/sol/sol.component.html:6]: added deviceId, authtoken, mpsServer

File: src/app/devices/sol/sol.component.spec.ts [hunk 2]
```diff
@@ -36,7 +33,7 @@ describe('SolComponent', () => {
         provide: ActivatedRoute,
         useValue:
           { params: of({ id: 'guid' }) }
-      }, { provide: AuthService, useValue: authService }],
+      }, { provide: AuthService, useValue: { getLoggedUserToken: () => '' } }],
```
Comment (rsdmike) [inline src/app/devices/sol/sol.component.spec.ts:36]: actually, the way this was done before was better, if you wanna change it back, and then update KVM test as well, that would be helpful

**Review by rsdmike — APPROVED** [review]

File: src/app/devices/sol/sol.component.spec.ts (hunk 2 above)
Comment (Mahendra-intel) [reply src/app/devices/sol/sol.component.spec.ts]: done. updated kvm test as well in kvm-integration branch

**Pull request merged**